<ui version="4.0" >
 <class>wallpaperFrm</class>
 <widget class="QWidget" name="wallpaperFrm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="2" >
    <spacer name="verticalSpacer_3" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="2" row="1" column="0" >
    <spacer name="horizontalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>13</width>
       <height>94</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="2" row="1" column="1" colspan="2" >
    <widget class="QLabel" name="current" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>140</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>140</width>
       <height>100</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text" >
      <string>TextLabel</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="1" column="3" >
    <spacer name="horizontalSpacer_3" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>13</width>
       <height>94</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4" colspan="2" >
    <widget class="QLabel" name="currentTitle" >
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" >
    <spacer name="verticalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="4" >
    <widget class="QListWidget" name="folders" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>180</width>
       <height>226</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="2" >
    <widget class="QListWidget" name="content" />
   </item>
   <item row="4" column="0" >
    <widget class="QToolButton" name="addBtn" >
     <property name="minimumSize" >
      <size>
       <width>27</width>
       <height>27</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>27</width>
       <height>27</height>
      </size>
     </property>
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>+</string>
     </property>
     <property name="icon" >
      <iconset resource="../syspref.qrc" >
       <normaloff>:/icons/tool/plus.png</normaloff>:/icons/tool/plus.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QToolButton" name="removeBtn" >
     <property name="minimumSize" >
      <size>
       <width>27</width>
       <height>27</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>27</width>
       <height>27</height>
      </size>
     </property>
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>-</string>
     </property>
     <property name="icon" >
      <iconset resource="../syspref.qrc" >
       <normaloff>:/icons/tool/minus.png</normaloff>:/icons/tool/minus.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>125</width>
       <height>19</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="4" >
    <spacer name="horizontalSpacer_4" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>358</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="5" >
    <widget class="QToolButton" name="helpBtn" >
     <property name="minimumSize" >
      <size>
       <width>27</width>
       <height>27</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>27</width>
       <height>27</height>
      </size>
     </property>
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>?</string>
     </property>
     <property name="icon" >
      <iconset resource="../syspref.qrc" >
       <normaloff>:/icons/tool/question.png</normaloff>:/icons/tool/question.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../syspref.qrc" />
 </resources>
 <connections/>
</ui>
